src/Raven.Client/Util/ConnectionStringParser.cs

```diff
@@ -89,17 +74,6 @@ protected virtual bool ProcessConnectionStringOption(ConnectionStringOptions opt
                 case "apikey":
                     options.ApiKey = value;
                     break;
-                case "user":
-                    networkCredentials.UserName = value;
-                    _setupUsernameInConnectionString = true;
-                    break;
-                case "password":
-                    networkCredentials.Password = value;
-                    _setupPasswordInConnectionString = true;
-                    break;
-                case "domain":
-                    networkCredentials.Domain = value;
-                    break;
                 case "urls":
```

Comment:
Why `urls`? Isn't it better to have a connection string like this:

`url=http://rvn-1:8080; url=http://rvn-2:8080; database=northwind` ?